src/components/PlanPage/PlanPage.js

```diff
@@ -11,15 +12,25 @@ class PlanPage extends React.Component {
     onStartSetActivities: PropTypes.func.isRequired,
   };
 
-  componentDidMount() {
-    this.props.onStartSetActivities();
+  constructor(props) {
```

Comment:
i'm pretty sure, that this project has proper configuration (you've been using arrow functions in class) to omit the constructor at all and do just:
`state = {....}` as a class property. 

The same thing is here:
`https://github.com/CharlieMantium/school-app/blob/master/src/components/ActivityForm/ActivityForm.js`
You could verify all places

Reply:
I don't remember which version of EcmaScript gave us this opportunity :P but this is a native possibility of newest javascript ;)